packages/auth/src/grant/routes.ts

```diff
@@ -413,6 +414,17 @@ async function continueGrant(
     accessToken
   })
 }
+async function deleteGrant(
+  deps: ServiceDependencies,
+  ctx: ContinueContext
+): Promise<void> {
+  const { id: continueId } = ctx.params
+  const deletion = await deps.grantService.deleteGrant(continueId)
+  if (deletion === 0) {
+    ctx.throw(404, { error: 'unknown_continue_id' })
+  }
+  ctx.status = 202
```

Comment:
```suggestion
  ctx.status = 204
```
https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol#section-5.4

Looks like oiur spec needs to be updated:
https://github.com/interledger/open-payments/blob/f2f84371778b61ec9481733b06d8d20a26ee9184/openapi/auth-server.yaml#L225

Reply:
* https://github.com/interledger/open-payments/pull/238